src/test/resources/cart-discount-with-relative-value.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
I don't see where this is used.

Reply:
I remove these ones in custom type related branch but make sense to remove it in this PR too.

Reply:
3c6ea89